graphite/graphite.go

```diff
@@ -100,6 +101,15 @@ func readTraceback(resp *http.Response) (*[]string, error) {
 // DefaultClient is the default HTTP client for requests.
 var DefaultClient = &http.Client{
 	Timeout: time.Minute,
+	Transport: &http.Transport{
+		MaxIdleConnsPerHost: 0,
```

Comment:
Why does setting that to 0 fix the problem? I see no mention of MaxIdle in those go issues.